ci-operator/config/openshift-kni/baremetal-deploy/openshift-kni-baremetal-deploy-master.yaml

```diff
@@ -0,0 +1,25 @@
+base_images:
+  base:
+    name: "4.3"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.12
+tag_specification:
+  name: "4.3"
+  namespace: ocp
+resources:
+  "*":
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+  - as: sctp-e2e
+    commands: cd features/sctp && make e2etest
```

Comment:
do we want to go ahead and create a top level "make e2etest" makefile target that is shared among multiple features? 

Ideally, i was thinking we could have a test suite that everyone shares across multiple features. What you have for sctp looks like a good foundation for that. We'd just need to separate the testsuite from the feature so it could contributed to further. 

thoughts?

Reply:
Having a global "make e2etest" sounds good. 
The only thing is, I'd prefer to have the relevant tests tied to each features path. That would make it easier to browse and understand the repo.

In that case though, we would have many suites, each with its "setup" phase that could lead to changes in the cluster which I am not sure is something we want. I am open for discussion :-)